src/index.js

```diff
@@ -29,7 +29,8 @@ export default class App extends React.Component {
     return (
       <Suspense fallback={null}>
         <Switch>
-          <Route path={`${path}/create`} component={JobCreateRoute} />
+          <Route path={`${path}/create/KBART`} component={JobCreateRoute} />
```

Comment:
So you don't need to define these Routes twice. You could actually just specify something like ``path={`${path}/create/:type?`}``. That would also give you the benefit of cleaning up all the if/else work you do in JobCreateRoute since `match.params.id` would then be `KBART` or `JSON`.

Reply:
Oops, looks like my comment here https://github.com/folio-org/ui-local-kb-admin/pull/81#discussion_r381322317 coincided with Marks comment here

Reply:
Have cleaned this up now.